ci-jobs/functional/run_appium.yml

```diff
@@ -0,0 +1,25 @@
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: Install Node 10.x
+- script: npm install -g appium
+  displayName: Install appium
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.x'
+- script: appium --relaxed-security &
```

Comment:
Could you move running appium after all check?

Reply:
You can also run appium server via Python script

Reply:
> You can also run appium server via Python script

Let me have another PR for this. I will think about how to use it and it(`Appium_service`) doesn't work well when I tried on my local..